src/scss/vars/_colors.scss

```diff
@@ -10,6 +10,7 @@
     --ons-color-grey-75: #707071;
     --ons-color-grey-35: #bcbcbd;
     --ons-color-grey-25: #d5d5d6;
+    --ons-color-grey-20: #e9eff4;
     --ons-color-grey-15: #e2e2e3;
     --ons-color-grey-10: #efeff0;
     --ons-color-grey-5: #f5f5f6;
```

Comment:
Comparing these colours this new one you've added is more of a blue grey so I dont think it fits with these colours. I would put it with the other blue grey colour and call it `--ons-color-light-blue-grey` or something

<img width="1720" alt="Screenshot 2025-04-08 at 10 16 12" src="https://github.com/user-attachments/assets/10a76c52-2294-4e0d-a4f2-5484f3fd6ddb" />